net.sf.eclipsecs.target/net.sf.eclipsecs.target.tpd

```diff
@@ -1,4 +1,5 @@
-// install http://mbarbero.github.io/fr.obeo.releng.targetplatform/p2/ to use this target definition
+// Install http://download.eclipse.org/cbi/tpd/3.0.0-SNAPSHOT/ to use this target definition.
+// Read https://github.com/eclipse-cbi/targetplatform-dsl for more details.
```

Comment:
There's some similar information in the readme.txt file that you may want to update.

Reply:
Done. I've taken the time to extend the documentation, such that also non eclipse developers may eventually understand what that target definition is used for.